Libplanet.Tests/Blockchain/Renderers/NonblockRendererTest.cs

```diff
@@ -97,5 +97,57 @@ Block<DumbAction> branchpoint
             // Double disposing does nothing.
             renderer.Dispose();
         }
+
+        [Theory]
+        [InlineData(4)]
+        [InlineData(8)]
+        [InlineData(16)]
+        [InlineData(32)]
+        public void StatingWorkerThreadOnlyOnce(int parallel)
+        {
+            // This is a regression test for the follwing bug:
+            // https://github.com/planetarium/libplanet/issues/1772
+            var renderer = new NonblockRenderer<DumbAction>(
+                new AnonymousRenderer<DumbAction>(),
+                25,
+                NonblockRenderer<DumbAction>.FullMode.DropNewest
+            );
+            Thread[] threads = new Thread[parallel];
```

Comment:
Why do we use `Thread` instead of `Task`? To my knowledge, we don't need to call low-level API `Thread` these days. IMO if you use `Task`, this test is way easier to read.

Plus, AFAIK `NineChronicles.Headless` is using `Renderer` with `Task`. So IMO we need to test like the user.

Reply:
> Why do we use `Thread` instead of `Task`? To my knowledge, we don't need to call low-level API `Thread` these days. IMO if you use `Task`, this test is way easier to read.

Because `Task`s are basically pooled.  In other words, they don't guarantee to start immediately, in particular when you spawn many tasks in a short time.  `Task`s can be spawned in a custom pool, but if we do that it's no more simpler than bare `Thread`… 🤔

> Plus, AFAIK `NineChronicles.Headless` is using `Renderer` with `Task`. So IMO we need to test like the user.

It does not intend to simulate *Nine Chronicles*' way to deal with renderers (by the way, does it really use renderers with tasks, and for what?), and it's even difficult to reproduce the race condition addressed by #1772 using tasks (due to the reason I wrote above).

Reply:
> Because Tasks are basically pooled. In other words, they don't guarantee to start immediately,

Ah, I miss that. 👍